monai/utils/module.py

```diff
@@ -214,7 +214,7 @@ def load_submodules(basemod, load_all: bool = True, exclude_pattern: str = "(.*[
     return submodules, err_mod
 
 
-def instantiate(path: str, **kwargs):
+def instantiate(__path: str, **kwargs):
```

Comment:
I would suggest to use "_path_" instead, follows other MONAI functions like:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/utils/module.py#L233
@wyli What do you think?

Thanks.

Reply:
I don't think it'll be part of content, so the current pr is better in my opinion

Reply:
Hi @surajpaib ,

I prefer to "`_path_`", if you think "`__path`" is better, I am also OK.
What do you think?

Thanks.

Reply:
Hi @Nic-Ma, I think since config attributes are named with the `_path_` convention, it might be nice to keep this different, hence submitted PR with `__path` based on discussion from #5865

Reply:
Since we also want non-conflicting paths, I think `__path` might be the most suitable. We could also switch to something like `_module_path` to prevent any potential conflict, although I don't see it being much of an issue.